### ISSUE TITLE
Does node:process.loadEnv support in bun

### ISSUE THREAD

**Issue by qiaoborui** [bot-base/telegram-bot-template#564]
looks like it cannot read the env file when I use the bun to run the application,
maybe because bun does not support to node:process to read envfile

**Comment by egeesenyel** [comment]
As far as I know bun has native support, the loadEnv function in bun is not needed because it should automatically load the file.

You can access the values in this way:
console.log(process.env.FOO)
console.log(Bun.env.SECRET)

So you may need to refactor the code that loads the configuration.

**Comment by qiaoborui** [comment]
> As far as I know bun has native support, the loadEnv function in bun is not needed because it should automatically load the file.
> 
> You can access the values in this way: console.log(process.env.FOO) console.log(Bun.env.SECRET)
> 
> So you may need to refactor the code that loads the configuration.

You're right, bun has native support for environment variables, but in this project node:process is used to read environment variables, if you use bun to run it you'll only get the empty environment variable
